source/assets/scripts/main.js

```diff
@@ -1,11 +1,119 @@
-// implement collapsible components for example Monday List
-var mondayList = document.getElementsByClassName("collapsible")[0];
-mondayList.addEventListener("click", (event) => {
-    var taskBoard = mondayList.nextElementSibling;
-    if (taskBoard.style.display == "none"){
-        taskBoard.style.display = "block";
+// main.js
+
+// Run the init() function when the page has loaded
+window.addEventListener("DOMContentLoaded", init);
+var dayTaskCount = new Map([
+    ["MondayId", 0],
+    ["TuesdayId", 0],
+    ["WednesdayId", 0],
+    ["ThursdayId", 0],
+    ["FridayId", 0],
+    ["SaturdayId", 0],
+    ["SundayId", 0]
+]);
+
+// Starts the program, all function calls trace back here
+function init() {
+    // add collapsible function to list titles
+    addCollapsibleControls();
+    addTasks();
+    let mondayFunc = document.getElementById("mFunc");
```

Comment:
In init(), You should call function rather than include specific implementation of function. You can follow what have you done with addTask() and create editTask() and deleteTake() here.